degoo/__init__.py

```diff
@@ -243,6 +244,11 @@ def login():
             
         print(f"No login credentials available. Please add account details to {cred_file}", file=sys.stderr)
 
+    if os.path.isfile(DP_file):
```

Comment:
A more canonical approach would be:

```
if not os.path.isfile(DP_file):
    source_file = os.path.basename(DP_file)
    if os.path.isfile(source_file):
        copyfile(source_file, DP_file)
    else:
        print(f"No properties are configured or available. If you can find the supplied file '{source_file}' copy it to '{DP_file}' and try again.")
```
Advantages being:

1. No dead conditional branch (`pass` is  a clue something might be tidier)
2. Avoids recoding the filename (configured at top)
3. Provides a useful tip if it goes completely wrong.

Reply:
Alright, I'll update the code and submit it for review ;)

Reply:
Updated 😉

Reply:
Not fussed about the number of commits but squashing them is fine. You did forget the "not" on line 247 ;-).

I like the idea of putting this in the login function (which you did)  as it isn't really relevant until after login.

Reply:
Hopefully this would be the last commit 😂